homeassistant/components/filter/sensor.py

```diff
@@ -488,7 +503,10 @@ def __init__(
     def _filter_state(self, new_state: FilterState) -> FilterState:
         """Implement the range filter."""
 
-        if self._upper_bound is not None and new_state.state > self._upper_bound:
+        # self._only_numbers = True
```

Comment:
left over ?

Reply:
No, I added them as comments to explain the need for casting.
Maybe I should adjust to a more meaningful comment...